indico/modules/events/registration/forms.py

```diff
@@ -357,7 +357,10 @@ class BadgeSettingsForm(IndicoForm):
                                  description=_("Display a dashed border around each badge"))
     page_size = IndicoEnumSelectField(_('Page size'), enum=PageSize)
     page_orientation = IndicoEnumSelectField(_('Page orientation'), enum=PageOrientation)
-    page_layout = IndicoEnumSelectField(_('Page layout'), enum=PageLayout)
+    page_layout = IndicoEnumSelectField(_('Page layout'), enum=PageLayout,
+                                        description=_('Single sided(foldable) option is only available if template '
+                                                      'orientation is the same as page orientation and its width is '
+                                                      'exactly half of the page width'))
```

Comment:
Do we really need this restriction? It makes it easier since people do not need to cut it but why not print it with cutting markers if it's smaller?